src/ProcessedStory.ts

```diff
@@ -33,17 +35,31 @@ export default class ProcessedStory {
         rawStory,
       ),
       timeout: normalizeTimeout(rawStory.parameters?.axe?.timeout, rawStory),
+      runOptions: normalizeRunOptions(
+        rawStory.parameters?.axe?.runOptions,
+        rawStory,
+      ),
     };
   }
 
+  // Determines if the test should be skipped in runSuite()
```

Comment:
👍 

Could make these JSDoc comments, so vscode shows them on hover

```suggestion
  /**
   * Determines if the test should be skipped in runSuite()
   */
```